src/deploy-cromwell-on-azure/scripts/startup.sh

```diff
@@ -112,6 +113,14 @@ kv["TesImageSha"]=$(docker inspect --format='{{range (.RepoDigests)}}{{.}}{{end}
 kv["TriggerServiceImageSha"]=$(docker inspect --format='{{range (.RepoDigests)}}{{.}}{{end}}' ${kv["TriggerServiceImageName"]})
 rm -f .env && for key in "${!kv[@]}"; do echo "$key=${kv[$key]}" >> .env; done
 
+write_log "Removing db lock if managed mysql"
+if [ -n "$mysql_server_name" ]; then
+    fully_qualified_server_name=$mysql_server_name.mysql.database.azure.com
+    mysql_server_password=${kv["MySqlServerPassword"]}
+    lockTableExists=$(mysql -u 'cromwell' -p$mysql_server_password cromwell_db -h $fully_qualified_server_name -P 3306 -e "SELECT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_schema = 'cromwell_db' AND table_name = 'DATABASECHANGELOGLOCK') AS '' ")
+    [[ $lockTableExists -eq '1' ]] && mysql -u 'cromwell' -p$mysql_server_password cromwell_db -h $fully_qualified_server_name -P 3306 -e "UPDATE cromwell_db.DATABASECHANGELOGLOCK SET LOCKED = 0, LOCKGRANTED = null, LOCKEDBY = null WHERE ID = 1" || :
```

Comment:
is this missing a space?  "-p$mysql_server_password"

Reply:
Surprisingly, it's not. @BMurri and I spent a lot of time debugging this one and adding a space messes it up.

Reply:
No. According to the command line reference on Oracle's website, a space there would result in two consequences: the password would be prompted for and expected to be received via stdin, and the actual password on the command line causes other command line parsing errors.